packages/technical-features/feature-core/README.md

```diff
@@ -0,0 +1,41 @@
+# @k8slens/feature-core
+
+Feature is set of injectables that are registered and deregistered simultaneously.
+
+## Install
+```bash
+$ npm install @k8slens/feature-core
+```
+
+## Usage
+
+```typescript
+import { createContainer } from "@ogre-tools/injectable"
+import { getFeature, registerFeature, deregisterFeature } from "@k8slens/feature-core"
+
+// Notice that this Feature is usually exported from another NPM package.
+const someFeature = getFeature({
+  id: "some-feature",
+  
+  register: (di) => {
+    di.register(someInjectable, someOtherInjectable);
+  },
+  
+  // Feature dependencies are automatically registered and 
+  // deregistered when necessary.
+  dependencies: [someOtherFeature] 
+});
+
+const di = createContainer("some-container");
+
+registerFeature(di, someFeature);
+
+// Or perhaps you want to deregister?
+deregisterFeature(di, someFeature);
+```
+
+## Need to know
+
+#### NPM packages exporting a Feature
+- Prefer `peerDependencies` since they are installed from the application and are not allowed to be in the built bundle. 
+- Prefer exporting `injectionToken` instead of `injectable` for not allowing other features to access technical details like the `injectable`
```

Comment:
Can the example above show this then. Not the best if the best practice is not shown in the only example code.

Reply:
That's just example which shows how Feature is created. We will have scaffolding for Features which will demonstrate both bullet points.